packages/vue/src/components/sign-in.vue

```diff
@@ -24,20 +30,18 @@ const forgotYourPasswordLink = computed(() =>
 const signInButtonText = computed(() => translate('Sign in'));
 const signIngButtonText = computed(() => translate('Signing in'));
 
-const { state, send } = useAuth();
-const actorState = computed(() =>
-  getActorState(state.value)
-) as ComputedRef<SignInState>;
+const { error, isPending, state, send, updateForm } = useAuthenticator();
+const { label, type } = getAliasInfoFromContext(state.context);
+const { formValues } = getActorContext(state) as SignInContext;
+const phoneNumber = computed(
+  () => `${formValues.country_code ?? ''}${formValues.phone ?? ''}`
```

Comment:
@ErikCH 

This isn't updating the `phoneNumber` in the view on change.

I suspect it's because the `formValues` from the machine aren't reactive